cypress/README.md

```diff
@@ -0,0 +1,30 @@
+    # Introduction
+
+This project is for End to end testing of Altinn solution using [cypress](https://www.cypress.io/).
+
+## Getting Started
+
+These instructions will get you run the end to end tests on test environments.
+
+### Install dependencies
+
+```cmd
+    npm install # only needed first time, or when dependencies are updated
+```
+
+### Open cypress and run tests
+
+Run the below command to open cypress with the environment, to run installation tests and regression tests
+
+```cmd
+    npm run cy:open --env=at22
+    
+```
+
+
+### Format files with prettier
+
+```cmd
+    npm run check # For checking the files deviating standards
```

Comment:
Does this folder use npm to be installed? If not this readme file needs to be updated. E.g. A lot of the commands uses npm instead of yarn